librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -1064,48 +1062,50 @@ commands:
         args:
           - name: format
             type: RZ_CMD_ARG_TYPE_STRING
-            optional: true
+            default_value: ' '
       - name: agi
         summary: Imports graph
         cname: analysis_graph_imports
         args:
           - name: format
             type: RZ_CMD_ARG_TYPE_STRING
-            optional: true
+            default_value: ' '
       - name: agr
         summary: References graph
         cname: analysis_graph_refs
         args:
           - name: format
             type: RZ_CMD_ARG_TYPE_STRING
-            optional: true
+            default_value: ' '
       - name: agR
         summary: Global references graph
         cname: analysis_graph_refs_global
         args:
           - name: format
             type: RZ_CMD_ARG_TYPE_STRING
-            optional: true
+            default_value: ' '
       - name: ags
         summary: Unknown graph
         cname: analysis_graph_unknown
-        args:
-          - name: num
-            type: RZ_CMD_ARG_TYPE_NUM
+        args: []
+      - name: agl
+        summary: Line graph ?
+        cname: analysis_graph_line
+        args: []
       - name: agx
         summary: Cross-references graph
         cname: analysis_graph_xrefs
         args:
           - name: format
             type: RZ_CMD_ARG_TYPE_STRING
-            optional: true
+            default_value: ' '
       - name: agg
         summary: Custom graph
         cname: analysis_graph_custom
         args:
           - name: format
             type: RZ_CMD_ARG_TYPE_STRING
-            optional: true
+            default_value: ' '
```

Comment:
Maybe make it `"normal"`? And have the default format called "normal". @ret2libc what do you think?

Reply:
@ret2libc is it possible to make a set of strings as the only possible values and autocomplete them?